continuous_integration/environment-mindeps.yaml

```diff
@@ -3,20 +3,20 @@ channels:
   - conda-forge
   - defaults
 dependencies:
-  - python=3.8
+  - python=3.9
   - click=8.0
   - cloudpickle=1.5.0
   - cytoolz=0.10.1
   - jinja2=2.10.3
   - locket=1.0.0
   - msgpack-python=1.0.0
   - packaging=20.0
-  - psutil=5.7.0
+  - psutil=5.7.2
```

Comment:
This is fine, but I'm curious what prompted this bump

Reply:
psutil isn't available for python 3.9 on conda-forge until 5.7.2